lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/domain/ReportProcessingResult.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.di.ipv.core.reportuseridentity.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.reportuseridentity.persistence.item.ReportUserIdentityItem;
+
+import java.util.List;
+import java.util.Map;
+
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@ExcludeFromGeneratedCoverageReport
+public record ReportProcessingResult(
+        ReportSummary summary,
+        List<ReportUserIdentityItem> users,
```

Comment:
I don't think there's any value in trying to return individual user info here - it'll either be too small to be of use, or too big to handle appropriately

Reply:
Removed